1) Em um concurso de arte com 15 participantes, as notas das avaliacoes dos quadros de cada um deles foram:

10, 6, 9, 8, 9, 8, 7, 7, 5, 10, 9, 8, 5, 3, 6.

No entanto, para a divulgacao dos resultados pelo marketing da galeria, sera necessario transformar esses valores nos conceitos A, B, C e D. O conceito D sera para notas abaixo de 5, conceito C para notas de 5 a 7, conceito B para notas de 7 a 9, e conceito A para notas acima de 9.

Calcule e construa uma tabela de distribuicao de frequencias para melhor visualizacao do desempenho dos concorrentes. 

In [ ]:
# Deixe seus calculos aqui:
lista_de_notas = [10, 6, 9, 8, 9, 8, 7, 7, 5, 10, 9, 8, 5, 3, 6]
minimo = min(lista_de_notas)
maximo = max(lista_de_notas)

print('Minimo: ', minimo)
print('Maximo: ', maximo)

Minimo:  3
Maximo:  10


In [ ]:
# calcular a amplitude
amplitude = int(maximo) - int(minimo)
print('amplitude: ', str(amplitude))

amplitude:  7


In [ ]:
# achar os intervalos = dividir o valor da amplitude pelo numero de classes
numero_de_classes = 4 # A B C D
intervalo = amplitude / numero_de_classes
print('Valor do intervalo: ', intervalo)

intervalo = 2

frequencia_d = sum(1 for item in lista_de_notas if item < 5)
print(frequencia_d)
frequencia_c = sum(1 for item in lista_de_notas if item >= 5 and item < 7)
print(frequencia_c)
frequencia_b = sum(1 for item in lista_de_notas if item >= 7 and item < 9)
print(frequencia_b)
frequencia_a = sum(1 for item in lista_de_notas if item >=9)
print(frequencia_a)

total = frequencia_d + frequencia_c + frequencia_b + frequencia_a
print('Total: ', total)

Valor do intervalo:  1.75
1
4
5
5
Total:  15


In [ ]:
frequencia_relativa_d = frequencia_d / len(lista_de_notas)
print(f'{(frequencia_relativa_d * 100):0.2f}')

frequencia_relativa_c = frequencia_c / len(lista_de_notas)
print(f'{(frequencia_relativa_c * 100):0.2f}')

frequencia_relativa_b = frequencia_b / len(lista_de_notas)
print(f'{(frequencia_relativa_c * 100):0.2f}')

frequencia_relativa_a = frequencia_a / len(lista_de_notas)
print(f'{(frequencia_relativa_a * 100):0.2f}')

6.67
26.67
26.67
33.33


| Nota  | Frequencia | Frequencia relativa |
| :---: | :----:     | :----: |
| 0 a 5 | 1          |  6.67 % |
| 5 a 7 | 4          |  26.67 % |
| 7 a 9 | 5          |  26.67 % |
| > 9   | 5          |  33.33 % |
| Total | 15         | 100%     |

2) Especifique os tipos das variaveis:

    a) peso de um lutador de sumo;
    b) marcas de motos;
    c) resistencia de materiais;
    d) quantidade de chuvas em Pocos de Caldas;
    e) nacionalidade;
    f) numero de versos em um poema;
    g) numero de acoes da Petrobras vendidas no dia;
    h) comprimento de terrenos;
    i) numero de mortes por ataques de tubarao no ano;
    j) resistencia a tracao de metais em kgf/mm^2;
    k) temperatura de ponto de ebulicao da agua, em graus Celsius;
    l) numero de itens nao conformes de uma linha de producao;
    m) tensao em Newtons de um fio de algodao esticado;
    n) numero de livros vendidos no mes.

In [ ]:
# Deixe seus calculos aqui:
# a) numerica continua;
# b) categorizada nominal;
# c) numerica continua;
# d) numerica continua;
# e) categorizada nominal;
# f) numerica discreta;
# g) numerica discreta;
# h) numerica continua;
# i) numerica discreta;
# j) numerica continua;
# k) numerica discreta;
# l) numerica discreta;
# m) numerica continua;
# n) numerica discreta;

3) Complete a tabela abaixo.

<h4 align="center"> Tabela 1: Tabela completa</h4>
    
| Nota | Frequencia  | Frequencia relativa | 
| :---: | :---: | :---: | 
| 9 a 10 | - | 0,08 | 
| 8 a 8,9 | 36 | - | 
| 6,5 a 7,9 | 90 | - | 
| 5 a 6,4 | 30 | - | 
| abaixo de 5 | 28 | - | 
| Total| 200 | 1,0 |

In [ ]:
# Deixe seus calculos aqui:
total = 200
todas_frequencias = 36 + 90 + 30 + 28
frequencia_nove_a_dez = todas_frequencias - total
print(abs(frequencia_nove_a_dez))

frequencia_relativa_linha_2 = 36 / total
print(f'{(frequencia_relativa_linha_2 * 100):0.2f} %')

frequencia_relativa_linha_2 = 90 / total
print(f'{(frequencia_relativa_linha_2 * 100):0.2f} %')

frequencia_relativa_linha_2 = 30 / total
print(f'{(frequencia_relativa_linha_2 * 100):0.2f} %')

frequencia_relativa_linha_2 = 28 / total
print(f'{(frequencia_relativa_linha_2 * 100):0.2f} %')

16
18.00 %
45.00 %
15.00 %
14.00 %


4) Em uma votacao com 3 candidatos, os votos foram:

Bruschi, Allan, Allan, Conte, Allan, Bruschi, Bruschi, Bruschi, Allan, Allan, Conte, Allan, Bruschi, Bruschi, Allan, Conte, Bruschi, Bruschi, Bruschi, Allan

a) Determine o numero de votos de cada pessoa;<br>
b) Calcule as porcentagens;<br>
c) Verifique se algum dos candidatos teve maioria absoluta (50% + 1).

In [ ]:
# Deixe seus calculos aqui:
votos = ['Bruschi', 'Allan', 'Allan', 'Conte', 'Allan', 'Bruschi', 'Bruschi', 'Bruschi', 'Allan', 
         'Allan', 'Conte', 'Allan', 'Bruschi', 'Bruschi', 'Allan', 'Conte', 'Bruschi', 'Bruschi', 
         'Bruschi', 'Allan']

quantidade_de_votos = len(votos)
print('Quantidade de votos: ', quantidade_de_votos)

# 3 Candidatos:
votos_para_bruschi = votos.count('Bruschi')
print('Votos para Bruschi: ', votos_para_bruschi)

votos_para_allan = votos.count('Allan')
print('Votos para Allan: ', votos_para_allan)

votos_para_conte = votos.count('Conte')
print('Votos para Conte: ', votos_para_conte)

procentagem_para_bruschi = (votos_para_bruschi / quantidade_de_votos) * 100
print('Porcentagem: ', procentagem_para_bruschi)
porcentagem_para_allan = (votos_para_allan / quantidade_de_votos) * 100
print('Porcentagem: ', porcentagem_para_allan)
procentagem_para_conte = (votos_para_conte / quantidade_de_votos) * 100
print('Porcentagem: ', procentagem_para_conte)

votos_no_total = votos_para_bruschi + votos_para_allan + votos_para_conte
print('Votos no total: ', votos_no_total)

if procentagem_para_bruschi > (50 + 1):
  print('Bruschi teve maioria absoluta!')
elif porcentagem_para_allan > (50 + 1):
  print('Allan teve maioria absoluta!')
elif procentagem_para_conte > (50 + 1):
  print('Conte teve maioria absoluta!')
else:
  print('Nenhum dos candidatos teve quantidade absoluta')

Quantidade de votos:  20
Votos para Bruschi:  9
Votos para Allan:  8
Votos para Conte:  3
Porcentagem:  45.0
Porcentagem:  40.0
Porcentagem:  15.0
Votos no total:  20
Nenhum dos candidatos teve quantidade absoluta


5) 40 doadores de sangue doaram para o hospital uma bolsa cada, conforme a tabela de tipos sanguineos:

B, A, O, A, A, A, B, O, B, A, A, AB, O, O, A, O, O, A, A, A, B, A, A, A, O, O, O, A, O, A, O, O, A, O, AB, O, O, A, AB, B, B.

Coloque os dados em uma tabela de distribuicao de frequencias.

In [ ]:
# Deixe seus calculos aqui:
tipos_sanguineos_dos_doadores = """B, A, O, A, A, A, B, O, B, A, A, AB, O, O, A,
O, O, A, A, A, B, A, A, A, O, O, O, A, O, A, O, O, A, O, AB, O, O, A, AB, B, B."""

tipos_sanguineos_dos_doadores = tipos_sanguineos_dos_doadores.replace('.', '').replace('\n', '').replace(' ', '')
tipos_sanguineos_dos_doadores = tipos_sanguineos_dos_doadores.split(',')
print(tipos_sanguineos_dos_doadores)

tipos_sanguineos = list(set(tipos_sanguineos_dos_doadores))
print(tipos_sanguineos)

frequencias = []

for tipo_sanguineo in tipos_sanguineos:
  frequencias.append(tipos_sanguineos_dos_doadores.count(tipo_sanguineo))

frequencia_total = sum(frequencias)
print('Frequencias :', frequencias)
print('Frequencia total: ', frequencia_total)

['B', 'A', 'O', 'A', 'A', 'A', 'B', 'O', 'B', 'A', 'A', 'AB', 'O', 'O', 'A', 'O', 'O', 'A', 'A', 'A', 'B', 'A', 'A', 'A', 'O', 'O', 'O', 'A', 'O', 'A', 'O', 'O', 'A', 'O', 'AB', 'O', 'O', 'A', 'AB', 'B', 'B']
['O', 'AB', 'A', 'B']
Frequencias : [15, 3, 17, 6]
Frequencia total:  41


In [ ]:
frequencia_relativa = []
for frequencia in frequencias:
  frequencia_relativa.append(frequencia / frequencia_total)

frequencia_relativa = list(map(lambda x: x * 100, frequencia_relativa))
print(frequencia_relativa)

[36.58536585365854, 7.317073170731707, 41.46341463414634, 14.634146341463413]


6) 80 alunos do curso de pos graducao tiveram o seguinte desempenho: 25% receberam nota A, 70% receberam nota B, e 5% tiveram nota C. Qual a frequencia dos alunos com nota A?

In [ ]:
# Deixe seus calculos aqui:
num_alunos_a = (25*80) / 100
num_alunos_a

7) Para que um determinado trecho em uma rota aeronautica seja lucrativo, um certo numero de assentos ocupados 
seria necessario. As classes de estudo para assentos desocupados foram: <br>
    0 a 4; 5 a 9; 10 a 14, 15 a 19, 20 ou mais.<br> 
    Com base nessas informacoes, seria possivel obter a quantidade de voos em que:
<br>
a) havia menos que 10 assentos desocupados?<br>
b) mais de 14 acentos desocupados?<br>
c) pelo menos 5 assentos desocupados?<br>
d) exatamente 9 assentos desocupados?

In [ ]:
# Deixe suas respostas aqui:
# a) sim, somar as requencias das 2 classes anteriores
# b) sim, somar frequencias das 2 classes posteriores
# c) sim, com excecao da primeira classe
# d) nao, pois temos os valores em intervalos

8) Em uma sala com fanaticos por motos, foram obtidos os seguintes dados para preferencias: 19 pessoas 
    gostariam de ter uma moto esportiva com carenagem, 13 desejam uma moto custom, 
    3 querem esportivas naked, 2 querem uma scooter, 2 preferem motos de trilhas, e 1 deseja um quadriciclo. 
    Monte uma tabela com distribuicao de frequencias.

In [ ]:
# Deixe seus calculos aqui:
dados = [19, 13, 3, 2, 2, 1]
total = sum(dados)

freq_rel_esportiva = dados[0] / total
print(freq_rel_esportiva)
freq_rel_custom = dados[1] / total
print(freq_rel_custom)
freq_rel_naked = dados[2] / total
print(freq_rel_naked)
freq_rel_scooter = dados[3] / total
print(freq_rel_scooter)
freq_rel_trilha = dados[4] / total
print(freq_rel_trilha)
freq_rel_quadriciclo = dados[5] / total
print(freq_rel_quadriciclo)

0.475
0.325
0.075
0.05
0.05
0.025


9) Uma competicao com 100 atletas, ha 7 com 1 medalha, 12 com 2 medalhas, 23 com 3 medalhas, 43 com 4 medalhas,
8 com 5 medalhas, 6 com 6 medalhas e 3 com 7 medalhas. Apresente os dados em uma tabela.

In [ ]:
# Deixe seus calculos aqui:


10) O setor de qualidade de uma fabrica de chocolates quer verificar se o peso de 65 gramas de um produto esta
de acordo com o informado na embalagem. Em uma busca aleatoria, obteve-se os seguintes dados:
    
<h4 align="center"> Tabela 3: Peso dos produtos, em gramas</h4>
  
| | | | |    
| :---: | :---: | :---: | :---: |
| 63,6 | 64,0 | 64,8 | 65,2 | 
| 63,7 | 64,0 | 64,8 | 65,3 |
| 63,7 | 64,2 | 64,9 | 65,3 | 
| 63,7 | 64,6 | 65,1 | 65,4 | 
| 63,7 | 64,7 | 65,2 | 65,4 | 


Construa uma tabela de distribuicao de frequencias.
    

In [95]:
import pandas as pd

# Deixe seus calculos aqui:
# calcular o intervalo
peso_dos_produtos_em_gramas = [63.6, 64.0, 64.8, 65.2, 63.7, 64.0,
64.8, 65.3, 63.7, 64.2, 64.9, 65.3, 63.7, 64.6, 65.1, 65.4, 63.7, 
64.7, 65.2, 65.4 ]

frequencias = []

for peso in peso_dos_produtos_em_gramas:
  frequencia_nao_encontrada = True

  for index, frequencia in enumerate(frequencias):
    if frequencia['Peso'] == peso:
      frequencias[index]['Frequencia'] = frequencias[index]['Frequencia'] + 1
      frequencia_nao_encontrada = False

  if frequencia_nao_encontrada:
    frequencias.append({'Peso': peso, 'Frequencia': 1})

frequencia_total = sum(frequencia['Frequencia'] for frequencia in frequencias)

for frequencia in frequencias:
  frequencia['Frequencia relativa'] = frequencia['Frequencia'] / frequencia_total

frequencias.append({'Peso': 'Total', 
                    'Frequencia': frequencia_total,
                    'Frequencia relativa': sum(frequencia['Frequencia relativa'] for frequencia in frequencias)})

# Exibir os dados em forma de tabela
df = pd.DataFrame(frequencias)

amplitude = int(max(peso_dos_produtos_em_gramas)) - int(min(peso_dos_produtos_em_gramas))
intervalo = amplitude / len(frequencias)
print('Valor do intervalo: ', intervalo)

df

Valor do intervalo:  0.15384615384615385


Peso  Frequencia  Frequencia relativa
0    63.6           1                 0.05
1      64           2                 0.10
2    64.8           2                 0.10
3    65.2           2                 0.10
4    63.7           4                 0.20
5    65.3           2                 0.10
6    64.2           1                 0.05
7    64.9           1                 0.05
8    64.6           1                 0.05
9    65.1           1                 0.05
10   65.4           2                 0.10
11   64.7           1                 0.05
12  Total          20                 1.00

11) Determine o tipo da variavel:


a) dados de contagem (discretos ou continuos);<br>
b) numero de vezes que um evento acontece (frequencia, frequencia relativa, total)<br>
c) cores de cabelos (ordinais, nominais, continuos, discretos).

In [ ]:
# Deixe suas respostas aqui:
# a) discreta
# b) frequencia
# c) nominais

12) Verdadeiro ou falso?
    
    a) frequencia relativa e igual a frequencia dividida pelo total.
    b) estatistica e a arte de apresentar graficos.

In [ ]:
# Deixe suas respostas aqui:
# a) verdadeiro
# b) falso. Estatística é a ciência que se utiliza das teorias probabilísticas para explicar a frequência da ocorrência de eventos

13) Para divulgacao de aprovados ou reprovados em uma disciplina, organize na forma de tabela de frequencias
as notas dos 15 alunos do curso da seguinte maneira: D para notas de 0 a 5 (reprovado); C para notas de 5 a 7;
B para notas de 7 a 9; e A para notas acima de 9.

dados: 7, 8, 6, 5, 6, 4, 7, 10, 7, 8, 8, 3, 8, 9, 10

In [94]:
# Deixe seus calculos aqui:
notas = [7, 8, 6, 5, 6, 4, 7, 10, 7, 8, 8, 3, 8, 9, 10]

frequencia_d, frequencia_c, frequencia_b, frequencia_a = 0, 0, 0, 0

for nota in notas:
  if nota > 0 and nota < 5:
    frequencia_d += 1
  elif nota >= 5 and nota < 7:
    frequencia_c += 1
  elif nota >= 7 and nota < 9:
    frequencia_b += 1
  else:
    frequencia_a += 1

total_frequencias = frequencia_d + frequencia_c + frequencia_b + frequencia_a

tabela_de_frequencias = [
  {'Intervalo': 'D para notas de 0 a 5 (reprovado)', 'Frequencia': frequencia_d},
  {'Intervalo': 'C para notas de 5 a 7', 'Frequencia': frequencia_c},
  {'Intervalo': 'B para notas de 7 a 9', 'Frequencia': frequencia_b},
  {'Intervalo': 'A para notas acima de 9', 'Frequencia': frequencia_a},
  {'Intervalo': 'Total', 'Frequencia': total_frequencias}
]

for linha in tabela_de_frequencias:
  linha['Frequencia Relativa'] = linha['Frequencia'] / total_frequencias

# Exibir os dados em forma de tabela
df = pd.DataFrame(tabela_de_frequencias)
df

Intervalo  Frequencia  Frequencia Relativa
0  D para notas de 0 a 5 (reprovado)           2             0.133333
1              C para notas de 5 a 7           3             0.200000
2              B para notas de 7 a 9           7             0.466667
3            A para notas acima de 9           3             0.200000
4                              Total          15             1.000000

14) O setor de qualidade de uma fabrica obteve a seguinte tabela com dados de embalagens com defeitos.

<h4 align="center"> Tabela 4: Estado de embalagens</h4>
  
| | | | | | 
| :---: | :---: | :---: | :---: | :---: |
| amassada | sem defeito  | sem defeito | amassada | sem defeito  |
| sem defeito  | cortada | amassada | sem defeito | sem defeito  |
| nao fechada | sem defeito  | sem defeito  | sem defeito  | sem defeito  |
| sem defeito | sem defeito | sem defeito | sem defeito  | sem defeito | 
| sem defeito  | sem defeito | sem defeito  | amassada | sem defeito | 

a) Construa uma tabela de distribuicao de frequencias;<br>
b) Explique se ha necessidade de melhorias no processo de embalagem.


In [93]:
# Deixe seus calculos aqui:
estado_das_embalagens = [ 'amassada','sem defeito','sem defeito','amassada','sem defeito'
,'sem defeito','cortada','amassada','sem defeito','sem defeito'
,'nao fechada','sem defeito','sem defeito','sem defeito','sem defeito'
,'sem defeito','sem defeito','sem defeito','sem defeito','sem defeito'
,'sem defeito','sem defeito','sem defeito','amassada','sem defeito' ]

frequencias = []

for peso in estado_das_embalagens:
  frequencia_nao_encontrada = True

  for index, frequencia in enumerate(frequencias):
    if frequencia['Peso'] == peso:
      frequencias[index]['Frequencia'] = frequencias[index]['Frequencia'] + 1
      frequencia_nao_encontrada = False

  if frequencia_nao_encontrada:
    frequencias.append({'Peso': peso, 'Frequencia': 1})

frequencia_total = sum(frequencia['Frequencia'] for frequencia in frequencias)

for frequencia in frequencias:
  frequencia['Frequencia relativa'] = frequencia['Frequencia'] / frequencia_total

frequencias.append({'Peso': 'Total', 
                    'Frequencia': frequencia_total,
                    'Frequencia relativa': sum(frequencia['Frequencia relativa'] for frequencia in frequencias)})

# Exibir os dados em forma de tabela
df = pd.DataFrame(frequencias)

amplitude = int(max(peso_dos_produtos_em_gramas)) - int(min(peso_dos_produtos_em_gramas))
intervalo = amplitude / len(frequencias)
print('Valor do intervalo: ', intervalo)

df

Valor do intervalo:  0.4


Peso  Frequencia  Frequencia relativa
0     amassada           4                 0.16
1  sem defeito          19                 0.76
2      cortada           1                 0.04
3  nao fechada           1                 0.04
4        Total          25                 1.00

b) A maioria dos defeitos é embalagem amassada, poderia ver nesse processo um melhoramento

15) O setor de qualidade da fabrica anterior resolveu verificar a eficiencia de duas turmas de trabalho.
A turma A teve uma producao de 563 itens conformes, e 151 itens nao conformes. A turma B produziu 307 itens conformes e 85 nao conformes. Organize os dados em uma tabela, mostrando a porcentagem de conformes e nao conformes.

In [103]:
# Deixe seus calculos aqui:
total_turma_a = 563 + 151
total_turma_b = 307 + 85
tabela = [
  {'Turma': 'A', 'Conformes': f'{((563 / total_turma_a) * 100):.2f} %', 'Nao conformes': f'{((151 / total_turma_a) * 100):.2f} %'},
  {'Turma': 'B', 'Conformes': f'{((307 / total_turma_b) * 100):.2f} %', 'Nao conformes': f'{((85 / total_turma_b) * 100):.2f} %' },
]

# Exibir os dados em forma de tabela
df = pd.DataFrame(tabela)
df

Turma Conformes Nao conformes
0     A   78.85 %       21.15 %
1     B   78.32 %       21.68 %

16) Analisando os dados de acidentes de trabalho no periodo de 3 anos da fabrica citada anteriormente, 
obteve-se a seguinte tabela:

<h4 align="center"> Tabela 5: Acidentes por departamento e dias da semana</h4>
  
| DEPARTAMENTO | segunda-feira | terca-feira | quarta-feira | quinta-feira | sexta-feira | 
| :---: | :---: | :---: | :---: | :---: | :---: |
| A | 12 | 21 | 23 | 29  | 35 |
| B  | 14 | 10 | 11 | 10 | 15 |
| C | 12  | 12  | 13 | 11 | 11 |
| D | 24 | 22 | 23 | 21 | 22 |

a) Em que dia da semana ocorreu mais acidentes?<br>
b) Em que departamento?

In [85]:
# Deixe seus calculos aqui:
dias_da_semana = [
  {'Dia': 'segunda-feira', 'Acidentes': 12 + 14 + 12 + 24 },
  {'Dia': 'terca-feira', 'Acidentes': 21 + 10 + 12 + 22 },
  {'Dia': 'quarta-feira', 'Acidentes': 23 + 11 + 13 + 23 },
  {'Dia': 'quinta-feira', 'Acidentes': 29 + 10 + 11 + 21 },
  {'Dia': 'sexta-feira', 'Acidentes': 35 + 15 + 11 + 22 }
]

maximo = dias_da_semana[0]['Acidentes']
dia_com_maior_numero_de_acidentes = dias_da_semana[0]['Dia']

for dia in dias_da_semana:
  if dia['Acidentes'] > maximo:
    maximo = dia['Acidentes']
    dia_com_maior_numero_de_acidentes = dia['Dia']

print('Dia da semana que ocorreu mais acidentes: ', dia_com_maior_numero_de_acidentes)

Dia da semana que ocorreu mais acidentes:  sexta-feira


In [87]:
departamentos = [
  {'Departamento': 'A', 'Acidentes': 12 + 21 + 23 + 29  + 35 },
  {'Departamento': 'B', 'Acidentes': 14 + 10 + 11 + 10 + 15 },
  {'Departamento': 'C', 'Acidentes': 12  + 12  + 13 + 11 + 11 },
  {'Departamento': 'D', 'Acidentes': 24 + 22 + 23 + 21 + 22 },
]

maximo = departamentos[0]['Acidentes']
departamento_com_maior_numero_de_acidentes = departamentos[0]['Departamento']

for dia in departamentos:
  if dia['Acidentes'] > maximo:
    maximo = dia['Acidentes']
    dia_com_maior_numero_de_acidentes = dia['Dia']

print('Departamento que ocorreu mais acidentes: ', departamento_com_maior_numero_de_acidentes)

Departamento que ocorreu mais acidentes:  A


17) A distribuicao dos salarios de 200 funcionarios, em multiplos de RS 1000,00, segue conforme a tabela abaixo:

<h4 align="center"> Tabela 6: Distribuicao de salarios de 200 funcionarios</h4>
  
| Classe | Porcentagem | 
| :---: | :---: | 
| 2 a 4 | 20 | 
| 4 a 5 | 25 | 
| 5 a 8 | 45  | 
| 9 a 10 | 10 | 
| Total | 100 | 

O numero de funcionarios que possuem salarios maiores ou iguais a RS 4000,00 e inferiores a RS 8000,00 e:

a) 60<br>
b) 80<br>
c) 90<br>
d) 140<br>
e) 160

In [92]:
# Deixe seus calculos aqui:
porcentagem_no_intervalo = (25 + 45) / 100
total_de_funcionarios = 200
numero_de_funcionarios =  total_de_funcionarios * porcentagem_no_intervalo
print('O numero de funcionarios que possuem salarios maiores ou iguais a RS 4000,00 e inferiores a RS 8000,00:')
print(numero_de_funcionarios)

O numero de funcionarios que possuem salarios maiores ou iguais a RS 4000,00 e inferiores a RS 8000,00:
140.0


18) Uma cidade de 10 mil habitantes gera aproximadamente 0,76 kg de lixo por pessoa em um dia, mas somente 82% e atendido
pelo aterro sanitario da cidade. A quantidade de lixo que realmente acaba indo para o aterro, em kg/dia, e: <br>

a) 1368; <br> 
b) 1558; <br>
c) 6232; <br>
d) 7600; <br>
e) 24928.

In [109]:
# Deixe seus calculos aqui:
numero_de_habitantes = 10_000
kilo_de_lixo_por_pessoa = 0.76
porcentagem_atendido_pelo_aterro = 0.82

total_kilos_de_lixo_por_dia = numero_de_habitantes * kilo_de_lixo_por_pessoa
quantidade_de_lixo_aterro = total_kilos_de_lixo_por_dia * porcentagem_atendido_pelo_aterro
print('A quantidade de lixo que realmente acaba indo para o aterro, em kg/dia, e:', quantidade_de_lixo_aterro)

A quantidade de lixo que realmente acaba indo para o aterro, em kg/dia, e: 6232.0


Resultado: c) 6232;